test/language/module-code/verify-dfs.js

```diff
@@ -0,0 +1,31 @@
+/*---
```

Comment:
we need a copyright header in each file (including the fixtures) for the lint to pass. This requirement is an general rule I'd love to remove but we can't change it for now.